### PR TITLE
Add links to Matrix and IRC chat when opening new issues

### PR DESCRIPTION
**Pull request by TobiGr**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [x] Meta improvement to the project (dev facing)

#### Description of the changes in your PR
Add links to our Matrix and IRC channels. They appear at the bottom when opening issues.

#### See in action
https://github.com/Chocobozzz/PeerTube/issues/new/choose

#### Due diligence
-  [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1 +1,8 @@
 blank_issues_enabled: false
+contact_links:
+  - name: 💬 IRC
+    url: https://webchat.freenode.net/#newpipe
+    about: Chat with us via IRC for quick Q/A here
+  - name: 💬 Matrix
+    url: https://matrix.to/#/#newpipe:matrix.org
```
Comment (mhmdanas) [inline .github/ISSUE_TEMPLATE/config.yml:7]: Shouldn't this be `#freenode_#newpipe:matrix.org`, or is there actually a separate Matrix room I'm unaware of?

Comment (TobiGr) [reply .github/ISSUE_TEMPLATE/config.yml]: good catch!
thanks

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1 +1,8 @@
 blank_issues_enabled: false
+contact_links:
+  - name: 💬 IRC
+    url: https://webchat.freenode.net/#newpipe
+    about: Chat with us via IRC for quick Q/A here
```
Comment (mhmdanas) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: Chat with us via IRC for quick Q/A
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -1 +1,8 @@
...
+  - name: 💬 Matrix
+    url: https://matrix.to/#/#freenode_#newpipe:matrix.org
+    about: Chat with us via Matrix for quick Q/A here
```
Comment (mhmdanas) [inline .github/ISSUE_TEMPLATE/config.yml:8]:
```suggestion
    about: Chat with us via Matrix for quick Q/A
```

**Review by mhmdanas — COMMENTED** [review]
Just two suggestions.

**Review by mhmdanas — APPROVED** [review]

**Pull request merged**